htdocs/core/modules/livraison/doc/pdf_typhon.modules.php

```diff
@@ -754,43 +754,6 @@ function _pagehead(&$pdf, $object, $showaddress, $outputlangs)
 
 		$posy+=2;
 
-		// Add list of linked orders on shipment
```

Comment:
If this is "duplicate", i can't find where is output first order reference of delivery ?